include/extractor/guidance/mergable_road_detector.hpp

```diff
@@ -77,10 +77,15 @@ class MergableRoadDetector
     // situations with more than two roads that could participate in a merge
     bool IsDistinctFrom(const MergableRoadData &lhs, const MergableRoadData &rhs) const;
 
-  private:
     // check if two name ids can be seen as identical (in presence of refs/others)
     // in our case this translates into no name announcement in either direction (lhs->rhs and
     // rhs->lhs)
+    static bool HaveIdenticalNames(const NameID lhs,
+                                   const NameID rhs,
+                                   const util::NameTable &name_table,
+                                   const SuffixTable &street_name_suffix_table);
+
+  private:
     bool HaveIdenticalNames(const NameID lhs, const NameID rhs) const;
```

Comment:
You could remove this wrapper function and replace it with direct calls to `HaveIdenticalNames` just to decrease the number of abstraction levels.